ui/src/pages/Cluster/New/index.tsx

```diff
@@ -103,11 +103,13 @@ export default function New() {
         style={{ marginBottom: 56 }}
       >
         <Row gutter={[16, 16]}>
-          <BasicInfo
-            passwordVal={passwordVal}
-            setPasswordVal={setPasswordVal}
-            form={form}
-          />
+          <Col span={24}>
```

Comment:
Why add `Col` here and delete `Col` somewhere else.

Reply:
Original writing:`<Row><Component /></Row> function Component(){ return <Col>....</Col> }` 
I think `Component` is a public component, and `Col` should be handled by its parent component.